### ISSUE TITLE
ExamController - Update

### ISSUE THREAD

**Issue by loxiuras** [melsaka/exam-project#9]
https://github.com/melsaka/exam-project/blob/e0f8586f88fd36f5ca4d19463e4e908ad825bb43/app/Http/Controllers/Api/ExamController.php#L57

`Model::update( $request->only( ...) ) `would do the same thing and is most of the time easier to read.

**Comment by winnie80** [comment]
> `Model::update( $request->only( ...) ) `would do the same thing and is most of the time easier to read.

Question: 
can one use something like `$exam->update( $request->validated() )` ?

**Comment by loxiuras** [comment]
@winnie80 , yes you can. I always use "only" with the specific column I want to save instead of everything in the validation array.
